hack/test-end-to-end-docker.sh

```diff
@@ -30,7 +30,17 @@ if [[ -z "${BASETMPDIR-}" ]]; then
   sudo rm -rf "${BASETMPDIR}"
   mkdir -p "${BASETMPDIR}"
 fi
-VOLUME_DIR="${BASETMPDIR}/volumes"
+
+# when selinux is enforcing, the volume dir selinux label needs to be
+# svirt_sandbox_file_t
+#
+# TODO: fix the selinux policy to either allow openshift_var_lib_dir_t
+# or to default the volume dir to svirt_sandbox_file_t.
+sudo mkdir -p /var/lib/openshift/openshift.local.volumes
```

Comment:
The `/var/lib/openshift` path doesn't exist on the host for pid 1 to see - it only exists in the container.